otherlibs/dynlink/natdynlink.ml

```diff
@@ -16,11 +16,15 @@
 
 (* Dynamic loading of .cmx files *)
 
+open Cmx_format
+
 type handle
 
-external ndl_open: string -> bool -> handle * bytes = "caml_natdynlink_open"
+type globals_map = (string * Digest.t * Digest.t * string list) list
```

Comment:
Fixed. I used the field names from the tuple's only use in `default_available_units`